velox/substrait/tests/substrait_virtualTable.json

```diff
@@ -0,0 +1,183 @@
+{
+ "extension_uris": [
```

Comment:
Let's move this file to 'data' subfolder for better separation of code and data.